writer/xls_writer.py

```diff
@@ -0,0 +1,108 @@
+import calendar
+
+import xlsxwriter
```

Comment:
`XlsWriter` will no longer be need if you use pandas. by using `DataFrame.to_excel`. [here](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_excel.html) you can find doc.